app/assets/stylesheets/trainees/course.scss

```
@@ -0,0 +1,37 @@
+@import '../colors';
+
+a:hover, :link {
+  text-decoration: none!important;
```

```suggestion
  text-decoration: none !important;
```
hạn chế dùng !important nhé a